cfme/services/catalogs/orchestration_template.py

```diff
@@ -237,7 +239,7 @@ class AddDialog(CFMENavigateStep):
     VIEW = AddDialogView
 
     def step(self):
-        self.view.configuration.item_select('Create Service Dialog from Orchestration Template')
+        cfg_btn("Create Service Dialog from Orchestration Template")
```

Comment:
Even if I add this:

class AddTemplateView(TemplateForm):

    add_button = Button("Add")
    configuration = Dropdown("Configuration")